documentation/src/main/asciidoc/ch11.asciidoc

```diff
@@ -800,83 +800,40 @@ entries from all the bundles with this name found on the classpath in addition t
 This mechanism is also helpful when creating large multi-module applications: Instead of putting all the constraint
 messages into one single bundle, you can have one resource bundle per module containing only those messages of that module.
 
-[[section-constraint-definition-contributor]]
-==== Constraint definitions via `ConstraintDefinitionContributor`
+[[section-programmatic-constraint-definition]]
+==== Constraint definitions via programmatic constraint declaration API
```

Comment:
"Adding constraint definitions programmatically"

The spec uses "constraint definition" for creating a new constraint type and "constraint declaration" for applying it to a property, method etc. So formally, with your change the API is now for "programmatic constraint definition and declaration". But that's a mouthful, so I wouldn't name it like that everywhere.